tests/templates/test_tensornetworks/test_MPS.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Tests for the MPS template.
+"""
+import pytest
+import random
+import numpy as np
+import pennylane as qml
+from pennylane.templates.tensornetworks.mps import *
+
+"""
+Unit tests for the MPS template.
+"""
+
+class TestIndicesMPS: 
+    """Test function that computes MPS indices"""
+    
+    def test_exception_wrong_dim(self):
+        """Verifies that exception is raised if the number of wires is incorrect."""
+        
+        n_wires = random.randrange(1,20,2)
```

Comment:
It's best to not use randomness in testing if possible. Let's change this to a fixed list of wire numbers